src/Athena/Views/Home/Main_Page.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Main_Page";
+}
+<br /><br /><br /><br /><br />
+
+<img src="~/images/UT-color-vert-trans.gif" class="responsive-img left" style="height:10em; max-width:100%" />
+<br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
```

Comment:
What is the purpose of these `<br/>`'s? Use padding or margins if you're trying to force spacing.